rethinkdb/datadog_checks/rethinkdb/queries.py

```diff
@@ -1,148 +1,149 @@
 # (C) Datadog, Inc. 2020-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-from . import operations
-from .document_db import DocumentQuery, transformers
 
-# System configuration.
-
-# See: https://rethinkdb.com/docs/system-tables/#configuration-tables
-config_summary = DocumentQuery(
-    source=operations.get_config_summary,
-    name='config_summary',
-    prefix='rethinkdb.config',
-    metrics=[{'type': 'gauge', 'path': 'servers'}, {'type': 'gauge', 'path': 'databases'}],
-    groups=[
-        {'type': 'gauge', 'path': 'tables_per_database', 'key_tag': 'database'},
-        {'type': 'gauge', 'path': 'secondary_indexes_per_table', 'key_tag': 'table'},
+ClusterMetrics = {
```

Comment:
Just to be clear about the changes and make sure we don't forget any metric here:
* `ClusterMetrics` = `config_summary`  metrics + `cluster_statistics` with tags
* `ServerMetrics` = `server_statistics` + `server_statuses` with tags
* `TableMetrics` + `DatabaseMetrics` = `config_summary` groups
* `DatabaseTableMetrics` = `table_statistics` metrics with tags + `table_statuses`
* `ReplicaMetrics` = `replica_statistics` with tags
* `ShardMetrics` = `table_statuses` enumerations with tags
* `JobMetrics` = `job_summary`
* `CurrentIssuesMetrics` = `current_issues_summary`

I think we're good here 👍  Just suggesting some renaming